models/models/ourdna.py

```diff
@@ -0,0 +1,73 @@
+from collections import defaultdict
+from typing import Any
+
+from pydantic import BaseModel
+
+
+class OurDNADashboard(BaseModel):
+    """Model for OurDNA Dashboard"""
+
+    collection_to_process_end_time: dict[str, int] = {}
+    collection_to_process_end_time_statistics: dict[str, float | None] = {}
+    collection_to_process_end_time_24h: dict[str, int] = {}
+    processing_times_by_site: dict[str, dict[int, int]] = defaultdict(
+        lambda: defaultdict(int)
+    )
+    total_samples_by_collection_event_name: dict[str, int] = defaultdict(int)
+    samples_lost_after_collection: dict[str, dict[str, Any]] = {}
+    samples_concentration_gt_1ug: dict[str, float] = {}
+    participants_consented_not_collected: list[int] = []
+    participants_signed_not_consented: list[int] = []
+
+    @staticmethod
+    def from_sample(d: dict) -> 'OurDNADashboard':
```

Comment:
I'm not really understanding the method name.
Is this method required, it obscures the names of the variables, where you can construct directly, and let pydantic do that heavy lifting for you. Pydantic will also throw an error if the types aren't right which I think is preferred behaviour.

If there are specific transformations you need to do here (it doesn't look like it), then for sure, but I'd collect with `**kwargs`, and only modify the fields you need modifying. Maybe worth us chatting about this when we catch-up!